theano/tensor/blas.py

```diff
@@ -962,7 +962,12 @@ class Gemm(GemmRelated):
     E_float = 'gemm requires floating-point dtypes'
 
     def __init__(self, inplace):
-        self.__setstate__({'inplace': inplace})
```

Comment:
I think this op is a good example to follow for the other op changed in this PR, except maybe for the _op_use_c_code.